bundles/alpha.model/src/alpha/model/util/Face.xtend

```diff
@@ -255,10 +263,13 @@ class Face {
 	 * from the union of saturated constraints.
 	 */
 	def toLinearSpace() {
-		// The constant terms of each constraint must be "removed" by setting them to 0.
+		// The constant terms and parameter coefficients of each constraint must be "removed" by setting them to 0.
 		val universe = ISLBasicSet.buildUniverse(space.copy)
+		val nbParams = space.dim(ISLDimType.isl_dim_param)
 		return saturatedConstraints
 			.map[c | c.copy.setConstant(0)]
+			.map[c | (0..<nbParams).map[i | c.copy.setCoefficient(ISLDimType.isl_dim_param, i, 0)]]
+			.flatten
```

Comment:
This should have been here all along. The following two faces (parallel faces of a cube) should have the same linear space:
```
f1 = [N]->{[i,j,k] : 0<=i,j,k<N and i=0}
f2 = [N]->{[i,j,k] : 0<=i,j,k<N and i=N}
```

Previously, the linear space of f2 would be `[N]->{[N,j,k]}` when it should be `[N]->{[0,j,k]}`. Now, it is correctly computed as `[N]->{[0,j,k]}`.